cwltool/main.py

```diff
@@ -414,8 +414,8 @@ def load_job_order(args, t, stdin, print_input_deps=False, relative_deps=False,
     if len(args.job_order) == 1 and args.job_order[0][0] != "-":
         job_order_file = args.job_order[0]
     elif len(args.job_order) == 1 and args.job_order[0] == "-":
-        job_order_object = yaml.load(stdin)
-        job_order_object, _ = loader.resolve_all(job_order_object, "")
+        job_order_object = yaml.load(stdin, yaml.RoundTripLoader)
```

Comment:
Should I change it back to `yaml.load(stdin, yaml.RoundTripLoader)`?
`yaml.round_trip_load()` causes an error in mypy check on Travis